spec/models/portfolio_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe Portfolio do
+  let(:portfolio)       { FactoryBot.create(:portfolio) }
+  let(:portfolio_item)  { FactoryBot.create(:portfolio_item) }
```

Comment:
Only do one space after the longest entry (I think the factories suffer the same problem)

(Note to self, get rubocop on here :smile:)

Reply:
@Fryguy - To be honest, I was hoping to see the Rubocop output, was interested if the rules for the 'new' hash style ( non hash rocket ) was okay.  So I didn't touch any of it.